ndarbas06/tobuloRadimas.php

```diff
@@ -0,0 +1,38 @@
+<?php
+    function dalikliuRadimas($s) {
+        $dal[] = NULL;
+        for ($i = 1; $i < $s; $i++) {
+            if ($s % $i === 0) {
+                if ($dal[0] === NULL) {
+                    $dal[0] = $i;
+                }
+                else {
+                    $dal[] = $i;
+                }
+            }
+        }
+        return $dal;
+    }
+
+    function tobulaSkaicius($s) {
+
+        $dal = dalikliuRadimas($s);
+        if (array_sum($dal) === $s) {
+          return $dal;
```

Comment:
O kam tu dalikliu grazinimas? Sita funkcija turetu grazinti true arba false. Daliklius mums grazina kita funkcija.

Reply:
Pradžioje norėjau išvesti skaičius ir daliklius.

Reply:
Galutiniame variante nederetu palikti to.